faq/prebid-server-faq.md

```diff
@@ -77,18 +77,94 @@ creates or updates the `uids` cookie.
 
 The most common source of requests for Prebid Server is from Prebid.js:
 
```

Comment:
Step 1 should be removed as it's not actually a step. I could be wrong but it sounds like a scenario description. I made a few tweaks to the rest of the content: 

The most common source of requests for Prebid Server is from Prebid.js in a scenario where the user doesn't have any cookies for the Prebid Server domain.
	1.	The user loads a page with Prebid.js that's going to call Prebid Server -- i.e. the pub has set up s2sConfig.
	2.	Immediately after confirming that s2sConfig is setup, Prebid.js calls Prebid Server's /cookie-sync endpoint to initiate syncing
	3.	Prebid Server determines there are no uids cookie and responds to the browser with a list of pixel syncs for bidders that need to be synced.
	4.	Prebid.js places all of the pixels on the page and initiates the auction.
	5.	Because the syncs haven't completed, the auction call to Prebid Server will not contain the uids cookie.
	6.	The first auction occurs without IDs
	7.	At some point later, the pixels come back to Prebid Server through a /setuid redirect, setting (or updating) the uids cookie.
	8.	The second page view will have the IDs available.

On #3 - Prebid Server determines there are no uids cookie - is uids an acronym or var name or is it uid cookies? If uids is correct then it should read: 

Prebid Server determines there is no uids cookie

or

Prebid Server determines there are no uids cookies